main.py

```diff
@@ -58,6 +59,24 @@ def flow_stats_by_dpid_flow_id(self, dpids):
                 flow_stats_by_id[dpid].update({flow_id: info_flow_as_dict})
         return flow_stats_by_id
 
+    def table_stats_by_dpid_table_id(self, dpids, table_ids):
+        """ Auxiliar funcion for v1/table/stats endpoint implementation.
+        """
+        table_stats_by_id = {}
+        tables_stats_dict_copy = self.tables_stats_dict.copy()
+        for dpid_dict, tables in tables_stats_dict_copy.items():
```

Comment:
`dpid_dict` name here is a bit of confusing since it's `table.switch.id`, can you rename as `dpid`?

Reply:
Sure